### PR TITLE
8319128: sun/security/pkcs11 tests fail on OL 7.9 aarch64

### PR DESCRIPTION
**Pull request by mpdonova**
In this PR, I included logic to skip tests on Oracle Linux prior to version 8.  The NSS binaries we are using for testing use a newer version of GLIBC than is included with OL 7.9.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8319128](https://bugs.openjdk.org/browse/JDK-8319128): sun/security/pkcs11 tests fail on OL 7.9 aarch64 (**Bug** - P3)

### Reviewers
 * [Matthias Baesken](https://openjdk.org/census#mbaesken) (@MBaesken - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/16547/head:pull/16547` \
`$ git checkout pull/16547`

Update a local copy of the PR: \
`$ git checkout pull/16547` \
`$ git pull https://git.openjdk.org/jdk.git pull/16547/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 16547`

View PR using the GUI difftool: \
`$ git pr show -t 16547`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/16547.diff">https://git.openjdk.org/jdk/pull/16547.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/16547#issuecomment-1799874176)

### CODE REVIEW

File: test/jdk/ProblemList.txt
Comment (mpdonova) [reply test/jdk/ProblemList.txt:632]: The shell scripts compiles Java files with additional flags and does some additional set up before executing the test.  I put the script on the ProblemList because the pkcs11 tests were failing on aarch64 and throwing a SkippedException from MultipleLogins.java would cause the test to fail. Ultimately, I could have made the change that is part of this PR (catching the SkippedException in the java file) but I didn't see it at the time.

**Comment by mpdonova** [comment]
Hello, I'm looking for a reviewer for this PR. Thanks!

**Review by MBaesken — APPROVED** [review]

File: test/jdk/sun/security/pkcs11/PKCS11Test.java
Comment (mpdonova) [reply test/jdk/sun/security/pkcs11/PKCS11Test.java]: That's a good idea; I moved the method to the Platform class.

File: test/jdk/sun/security/pkcs11/PKCS11Test.java
Comment (mpdonova) [reply test/jdk/sun/security/pkcs11/PKCS11Test.java]: Yes, I think the method would also return true for RH 7.

**Comment by mpdonova** [comment]
Hello, this PR was approved but then I had to make some additional changes so I'd like to have an additional review/approval before merging it.

Thanks!

File: test/lib/jdk/test/lib/Platform.java
Comment (MBaesken) [inline test/lib/jdk/test/lib/Platform.java:354]: Hi,  I checked and the el ("Enterprise Linux" ?)  is shown in os.version on RHEL and Oracle Linux.  But how reliable is it? Could this show up also on some other distros in os.version (maybe in the cpu-related string of uname output?) ?

Comment (mpdonova) [reply test/lib/jdk/test/lib/Platform.java:354]: I'm not aware of any conventions in the output of uname that would prevent other operating systems from including "el" followed by a number.

Comment (MBaesken) [reply test/lib/jdk/test/lib/Platform.java:354]: okay , then let's keep it the way it is now and adjust later if needed.

**Review by MBaesken — APPROVED** [review]

**Pull request closed without merging**
